.rubocop.yml

```diff
@@ -26,6 +26,7 @@ AllCops:
     - 'Vagrantfile'
     - 'vendor/**/*'
     - 'lib/json_ld/*'
+    - 'lib/mastodon/migration_helpers.rb' # Vendored from GitLab
```

Comment:
This approach is fine, but another idea - move the file to somewhere under `vendor/`.

- That dir is already ignored by rubocop
- Since the file is copy/pasted from an external source, that's an appropriate home for it

Would just need to make sure load paths were correct so it was still usable.

For bonus points - pull in the new version of the file and adjust any changed method usage, etc.

Reply:
Good points. I don't think I'd be comfortable that if I changed that, i'd get it right. Feel free to submit a replacement PR, and I can close this one